mappings/net/minecraft/client/gui/screen/ingame/HandledScreen.mapping

```diff
@@ -94,3 +94,4 @@ CLASS net/minecraft/class_465 net/minecraft/client/gui/screen/ingame/HandledScre
 		ARG 1 x
 		ARG 2 y
 		ARG 3 color
+	METHOD method_37432 tickHandledScreen ()V
```

Comment:
Why can't this just be `tick`? If this is called as a specific tick method from a general tick method, consider `handledScreenTick` or `internalTick` etc (I think we have an `Entity.baseTick`, or it may be in `LivingEntity`)

Reply:
I named it like that since `tick` would conflict with `Screen.tick`, which is now declared final in `HandledScreen`

Reply:
I suggest `handledScreenTick` as the handled screen part of the tick is the object itself than a member of this handled screen. Feel free to find a more concise name.